cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/TerminateStackHandler.java

```diff
@@ -67,6 +67,7 @@ public void accept(Event<TerminateStackRequest> terminateStackRequestEvent) {
             } else {
                 result = new TerminateStackResult(request);
             }
+            connector.credentials().delete(ac);
```

Comment:
Is it not necessary to check the returned CloudCredentialStatus? It can be failed.